app/views/organizations/edit.html.erb

```diff
@@ -100,6 +102,27 @@
 
                 <%= f.input :repackage_essentials, label: 'Does your bank repackage essentials?', as: :radio_buttons, collection: [[true, 'Yes'], [false, 'No']], label_method: :second, value_method: :first %>
                 <%= f.input :distribute_monthly, label: 'Does your bank distribute monthly?', as: :radio_buttons, collection: [[true, 'Yes'], [false, 'No']], label_method: :second, value_method: :first %>
+
+                <% if Flipper.enabled?(:enable_packs) %>
+                  <%= label_tag "organization[request_unit_names]", 'Custom request units used (please use singular form -- e.g. pack, not packs)' %>
+                  <%= select_tag(
```

Comment:
This whole block is kinda ugly and I welcome feedback. I did a raw select tag so that I could give it a field name that isn't part of the model -- `request_unit_names`. It also uses the name as both the value and the display text. The rest is configuration for the select2.

Reply:
Would we reuse this type of UI anywhere else? Would it make sense to put it e.g. in `ui_helpers`?

Reply:
Maybe but I think not worth it until that happens. I might be biased since I wanna be done with this PR :)